locales/ar.json

```diff
@@ -1,689 +1,487 @@
-{
-  "org": {
-    "vue": {
```

Comment:
Some of the updates reintroduce the outdated key hierarchy (without the `org.vuejs` prefix) which I had fixed manually:

![grafik](https://user-images.githubusercontent.com/495429/46422084-d6743680-c733-11e8-969e-f1db0dc2dea7.png)

Those are the ones with a lot of lines added and removed:

![grafik](https://user-images.githubusercontent.com/495429/46421929-872e0600-c733-11e8-93db-7a32272f72f3.png)

Reply:
sorry about that! should be fixed now